examples/postgres/test.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+set -x
+
+./build.sh
+
+# test it
+docker stop test-postgres-func
+docker rm test-postgres-func
+
+docker run -p 5432:5432 --name test-postgres-func -d postgres
+sleep 5s
+
+docker run --rm -i --link test-postgres-func:postgres postgres psql -h postgres -U postgres -c 'CREATE TABLE people (first TEXT, last TEXT, age INTEGER);'
+
+RECORD1='{
+    "first": "John",
+    "last": "Smith",
+    "age": 30
+}'
+echo $RECORD1 | docker run --rm -i -e SERVER=postgres:5432 -e COMMAND=INSERT -e TABLE=people --link test-postgres-func:postgres iron/func-postgres
```

Comment:
I like both of these changes/fixes (adding --link and fixing bug). If you make a new PR separate from this one, that would be great!